.vib/logstash/goss/logstash.yaml

```diff
@@ -0,0 +1,11 @@
+file:
```

Comment:
Checking our compilation logic, I think we should check the `/opt/bitnami/logstash/jdk` folder does indeed not exist.

Reply:
Good point, adding it.